### PR TITLE
[Android] Small performance fixes to ListViewRenderer, PlatformSpecific IsFastScrollEnabled 

### PR DESCRIPTION
**Pull request by rmarinho**
### Description of Change ###

Work on some feedback from @Clancey  for ListViewRenderer ,

- Add Platform specific for IsFastScrollEnabled 
- Cache count

### API Changes ###

Added:
 - Platform specific `IsFastScrollEnabled `

Usage:
- `listView.On<Android>().SetIsFastScrollEnabled(true);`

### PR Checklist ###

- [ ] Has tests (if omitted, state reason in description)
- [x] Rebased on top of master at time of PR
- [x] Changes adhere to coding standard
- [x] Consolidate commits as makes sense

### CODE REVIEW

File: Xamarin.Forms.Core/PlatformConfiguration/AndroidSpecific/ListView.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
+﻿namespace Xamarin.Forms.PlatformConfiguration.AndroidSpecific
+{
+	using FormsElement = Forms.ListView;
+
+	public static class ListView
+	{
+		public static readonly BindableProperty IsFastScrollEnabledProperty =
+				BindableProperty.Create("IsFastScrollEnabled", typeof(bool),
+				typeof(ListView), true);
```
Comment (StephaneDelcroix) [inline Xamarin.Forms.Core/PlatformConfiguration/AndroidSpecific/ListView.cs]: are we sure about `true` as default value ?

Comment (rmarinho) [inline Xamarin.Forms.Core/PlatformConfiguration/AndroidSpecific/ListView.cs]: thanks, fixed!

**Comment by samhouts** [comment]
Some tests failing, @rmarinho.

**Comment by rmarinho** [comment]
@samhouts , @StephaneDelcroix ready for another review.

**Pull request merged**